test/algorithms/simulation/sidb/detect_bdl_pairs.cpp

```diff
@@ -0,0 +1,123 @@
+//
+// Created by marcel on 21.07.23.
+//
+
+#include <catch2/catch_template_test_macros.hpp>
+
+#include <fiction/algorithms/simulation/sidb/detect_bdl_pairs.hpp>
+#include <fiction/io/print_layout.hpp>
```

Comment:
I think this header is redundant.

Reply:
Good catch! That's indeed a leftover from testing.